app/src/logging/presidio_pii_filter.py

```diff
@@ -82,6 +84,20 @@ def _redact_string(self, value: str) -> str:
                 text=value, entities=self._entities, language="en"
             )
 
+            # Filter out emails from allowed domains
+            filtered_results = []
+            for field in fields_for_redaction:
+                # If it's an email, check if it's from a domain we aren;t redacting
+                if field.entity_type == "EMAIL_ADDRESS":
+                    email_text = value[field.start : field.end]
+                    # If email ends with any value not in the list of domains to skip, add to list for redaction
+                    if not any(
+                        email_text.endswith("@" + domain) for domain in email_skip_redaction_list
+                    ):
```

Comment:
For better readability, refactor this into a separate function called `email_address_in_skiplist` and use it like this:
```python
filtered_results = [field for field in fields_for_redaction if not email_address_in_skiplist(field)]
```